src/Client.php

```diff
@@ -264,6 +269,8 @@ private function configureDefaults(array $config): void
             $this->config['headers'] = ['User-Agent' => Utils::defaultUserAgent()];
         } else {
             // Add the User-Agent header if one was not already set.
+
+            /** @var string $name */
```

Comment:
Similar to my other comment in `CurlFactory`, since there is no array-shape here, the value of `$name` will be `array-key` (`int|string`), so PHPStan thinks we're passing `int` to `\strtolower()`.